### ISSUE TITLE
Gray text in table CTA is not accessible

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#1247]
The gray text in table CTA such as the table below doesn't meet accessible color contrast standards.
![image](https://user-images.githubusercontent.com/9530293/155013705-9d939168-b282-4b65-b9a8-d3b54e392ff5.png)

Even with [WCAG AA standards](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) for a small text like CTA text, the contrast needs to at least be 4.5:1, and right now it's 3.23:1

**Comment by harrislapiroff** [comment]
Let's futz with the design and pick a gray that's 4.5:1 and see how it looks
